includes/utilities.inc

```diff
@@ -66,6 +66,10 @@ function islandora_openseadragon_construct_clip_url($params, $download = FALSE)
   }
   $rft_id = $decoded_params['rft_id'];
   $path = parse_url($rft_id, PHP_URL_PATH);
+  // Remove sub dir from $path
```

Comment:
@simonhm Travis really likes periods, could you add one to the end of this comment 😄